janus_collector/src/lib.rs

```diff
@@ -19,10 +20,10 @@
 //! // Supply DAP task paramenters.
 //! # let task_id = TaskId::random();
 //! # let (hpke_config, private_key) = janus_core::hpke::generate_hpke_config_and_private_key(
-//! #     janus_core::message::HpkeConfigId::from(0),
```

Comment:
Given the changes here and in the use directive above, we should probably drop these `#` comments to show the reader what we're doing with generate_hpke_config_and_private_key and the rest.